py-kms/pykms_WinService.py

```diff
@@ -0,0 +1,55 @@
+# Warning: this script only for Windows 7 SP1 / Windows 2008 R2!
+# -
+# Download links x64:
+# https://www.python.org/ftp/python/3.7.4/python-3.7.4-amd64.exe
+# https://github.com/mhammond/pywin32/releases/download/b224/pywin32-224.win-amd64-py3.7.exe
+# -
+# Download links x32:
+# https://www.python.org/ftp/python/3.7.4/python-3.7.4.exe
+# https://github.com/mhammond/pywin32/releases/download/b224/pywin32-224.win32-py3.7.exe
+# -
+# Need install Python 3.7.4 before pywin32!
+# -
+# 1. Install Python 3.7.4 into C:\Windows\Python374
+# 2. Install pywin32
+# 3. Run cmd as Administrator and start command "cd C:\Windows\Python374\py-kms"
+# 4. Install py-kms as service command: "python pykms_WinService.py --startup delayed install" 
+# -
+import win32serviceutil
+import win32service
+import win32event
+import servicemanager
+import socket
+import subprocess
+
+class AppServerSvc (win32serviceutil.ServiceFramework):
+    _svc_name_ = "py-kms"
+    _svc_display_name_ = "py-kms"
+    _proc = None
+    _cmd = ["C:\Windows\Python374\python.exe", "C:\Windows\Python374\py-kms\pykms_Server.py"]
```

Comment:
Why is the path to this file hard coded to the local installation directory of Python 3.7.4 on Windows?

Reply:
If the paths are not strictly specified, then it does not work

Reply:
Sorry, that must be resolved. Try checking for some environment variable, which points to the correct Python version.

Reply:
I agree with @simonmicro, this must be configurable. Not everyone (and hopefully no one) will install py-kms to their Python install directory.